topics/transcriptomics/tutorials/unaligned-ncrnas-analysis/tutorial.md

```diff
@@ -0,0 +1,502 @@
+- Learn about the tools used in the workflow
+time_estimation: 1H
+key_points:
+- Homology
+- RNA
+- Secondary structure
+- Coding potential
+contributions:
+  authorship:
+    - eggzilla
+    - bebatut
+  editing:
+    - pavanvidem
+---
+
+# Introduction
+
+Non coding RNAs (ncRNA) that are homologous, meaning that they share a common ancestor RNA, can be identified
+by sharing a common sequence or structure. Since the biological function depends on the
+spatial structure of a molecule, it is often strongly conserved for ncRNAs.
+
+This workflow investigates a set of sequences for their sequence and structure conservation and for
+their alignability. We will inspect a set of dual function RNAs and a set of epidermal growth factor sequences with the workflow and compare the different outcomes.
+
+The first step in any analysis of unaligned ncRNAs is to run a multi alignment tool on the sequences of the ncRNAs. Using the multi alignment, we can extract some representative sequences, analyze them for protein coding potential, compute consensus structure and predict stable secondary structure.
+
+For the multi-alignment, two types of tools are used: one generic multi-aligner and one RNA-secondary structure aware.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Get data
+
+We will retrieve two files one file containing homolog sequences from the
+SgrS family, which has been discovered to be a dual-function RNA.
+This allows us to show the ability of the workflow to test for protein
+coding potential, as well as for predicting functional non-coding RNAs.
+
+
+> <hands-on-title> Data upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the files from [Zenodo](https://zenodo.org/record/3482616) or from the shared data library
+>
+>    ```
+>    {{page.zenodo_link}}/files/RF00534.fasta
+>    {{page.zenodo_link}}/files/GCF_000005845.2_ASM584v2_genomic.fna
+>    ```
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. {% icon galaxy-pencil %} **Rename** the datasets
+>    - Rename `reGCF_000005845.2_ASM584v2_genomic.fna` to `genome.fa`
+>    - Rename `RF00534.fasta` to `sgrs.fa`
+>
+>    {% snippet faqs/galaxy/datasets_rename.md %}
+>
+> 4. Check the datatypes for both files
+>    - Both files should automatically be detected as `fasta` format files.
+>    - If this is not the case, please manually set the datatype
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="fasta" %}
+>
+>
+{: .hands_on}
+
+# Analysis using generic multi alignment tool
+
+## Align sequences and extract representative sequences
+
+> <hands-on-title> Run multi-alignment tool and extract representative sequences from the alignment </hands-on-title>
+>
+> 1. **MAFFT** {% icon tool %} with the following parameters:
```

Comment:
@pavanvidem I've tried updating the tutorial to the latest GTN standards, but if you could just update the hands-on boxes to list the tools like below, at whichever version of the tool you use for the tutorial, then it will be compatible with GTN-in-Galaxy :) Thanks!

```suggestion
> {% tool [MAFFT](toolshed.g2.bx.psu.edu/repos/rnateam/mafft/rbc_mafft/7.508+galaxy1) %}  with the following parameters:
```

Reply:
sure, I will do that